tests/expressions_window/tests.py

```diff
@@ -406,23 +407,25 @@ def test_ntile(self):
     def test_percent_rank(self):
         qs = WindowTestModel.objects.annotate(percent_rank=Window(
             expression=PercentRank(),
-            order_by='salary')
-        ).order_by('percent_rank', 'name')
+            order_by=ExpressionList(OrderBy(F('salary')), OrderBy(F('name')))),
+        ).order_by('percent_rank')
 
+        # round for test unification, Oracle and PostgreSQL have a different number
+        # of decimal places
         self.assertQuerysetEqual(qs, [
             ('Moore', 'IT', 34000, 0.0),
-            ('Williams', 'Accounting', 37000, 0.0909090909090909),
-            ('Smith', 'Marketing', 38000, 0.181818181818182),
-            ('Johnson', 'Marketing', 40000, 0.272727272727273),
-            ('Jenson', 'Accounting', 45000, 0.363636363636364),
-            ('Jones', 'Accounting', 45000, 0.363636363636364),
-            ('Adams', 'Accounting', 50000, 0.545454545454545),
-            ('Brown', 'Sales', 53000, 0.636363636363636),
-            ('Smith', 'Sales', 55000, 0.727272727272727),
-            ('Wilkinson', 'IT', 60000, 0.818181818181818),
-            ('Johnson', 'Management', 80000, 0.909090909090909),
+            ('Williams', 'Accounting', 37000, 0.09090909),
+            ('Smith', 'Marketing', 38000, 0.18181818),
+            ('Johnson', 'Marketing', 40000, 0.27272727),
+            ('Jenson', 'Accounting', 45000, 0.36363636),
+            ('Jones', 'Accounting', 45000, 0.45454545),
+            ('Adams', 'Accounting', 50000, 0.54545455),
+            ('Brown', 'Sales', 53000, 0.63636364),
+            ('Smith', 'Sales', 55000,  0.72727273),
+            ('Wilkinson', 'IT', 60000, 0.81818182),
+            ('Johnson', 'Management', 80000, 0.90909091),
             ('Miller', 'Management', 100000, 1.0),
-        ], transform=lambda row: (row.name, row.department, row.salary, row.percent_rank),)
+        ], transform=lambda row: (row.name, row.department, row.salary, round(row.percent_rank, 8)),)
```

Comment:
Thanks. Adjusted and pushed. When you started the tests on Oracle, it was possible to see the test failures. Could you please do that again? There are a couple of tests (aggregation and the `RANGE BETWEEN n PRECEDING AND n FOLLOWING` that still needs to be filled out). Again, I really appreciate the input/help :)